packages/vscode-extension/package.nls.json

```diff
@@ -472,5 +472,10 @@
     "teamstoolkit.chatParticipants.create.selectFolder.label": "Select Folder",
     "teamstoolkit.chatParticipants.create.successfullyCreated": "Project successfully created in current workspace.",
     "teamstoolkit.chatParticipants.create.failToCreate": "Unable to create the project.",
-    "teamstoolkit.chatParticipants.default.noConceptualAnswer": "This is a procedural question, @teams can only answer questions regarding descriptions or concepts for now. You could try these commands or learn more from [Teams Toolkit documentation](https://learn.microsoft.com/microsoftteams/platform/toolkit/teams-toolkit-fundamentals).\n\n • /create: Use this command to find relevant templates or samples to build your Teams app as per your description. E.g. @teams /create create an AI assistant bot that can complete common tasks.\n\n • /nextstep: Use this command to move to the next step at any stage of your Teams app development."
+    "teamstoolkit.chatParticipants.default.noConceptualAnswer": "This is a procedural question, @teams can only answer questions regarding descriptions or concepts for now. You could try these commands or learn more from [Teams Toolkit documentation](https://learn.microsoft.com/microsoftteams/platform/toolkit/teams-toolkit-fundamentals).\n\n • /create: Use this command to find relevant templates or samples to build your Teams app as per your description. E.g. @teams /create create an AI assistant bot that can complete common tasks.\n\n • /nextstep: Use this command to move to the next step at any stage of your Teams app development.",
+    "teamstoolkit.chatParticipants.officeAddIn.description": "Use this command to ask questions about Office Add-ins development.",
+    "teamstoolkit.chatParticipants.officeAddIn.create.description": "Use this command to build your Office Add-ins as per your description.",
+    "teamstoolkit.chatParticipants.officeAddIn.generateCode.description": "Use this command to generate code for the Office add-in.",
+    "teamstoolkit.chatParticipants.officeAddIn.harmfulInputResponse": "Sorry, I can't help with that.",
+    "teamstoolkit.chatParticipants.officeAddIn.default.noConceptualAnswer": "This is a procedural question, @office can only answer questions regarding descriptions or concepts for now. You could try these commands or learn more from [Office Add-ins documentation](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins).\n\n • /create: Use this command to build your Office Add-ins as per your description. \n\n • /generatecode: Use this command to generate code for the Office add-in. \n\n • /nextstep: Use this command to move to the next step at any stage of your Office Add-in development."
```

Comment:
Change 'learn more' to 'get more info'.
Change 'office add-in' to 'office add-ins'